README.md

```diff
@@ -163,8 +163,11 @@ index 0e3a6a705d..416b8b0fb9 100644
   This will return a new version name, make sure it matches what you expect
 - Generate the changelog content from the [release](https://github.com/nextcloud-libraries/nextcloud-vue/releases) page.
   Create a draft release, select the previous tag, click `generate` then paste the content to the `CHANGELOG.md` file
-  1. use the the version as tag AND title (e.g `v4.0.1`)
-  2. add the changelog content as description (https://github.com/nextcloud-libraries/nextcloud-vue/releases)
+  1. adjust the links to the merged pull requests and authors so that the changelog also works outside of GitHub:
+     Apply this regex: `by @([^ ]+) in ((https://github.com/)nextcloud-libraries/nextcloud-vue/pull/(\d+))`
+     Which this as the replacement: `[\#$4]($2) \([$1]($3$1)\)`
```

Comment:
I feel like an additional manual step will increase inconsistency instead...

Is it a problem is username is not a link? The PR is a link, where author is als available.

Reply:
> Is it a problem is username is not a link? The PR is a link, where author is als available.

Yes author is not a valid link and also its pretty ugly when seen outside of GitHub (where you probably do not read it because there is the releases tab).
But you are right, its unnecessary work - so I created a script for it.